openedx/core/lib/xblock_builtin/xblock_discussion/static/js/mathjax_include.js

```diff
@@ -0,0 +1,36 @@
+var vendorScript;
```

Comment:
@itsjeyd Thanks for the detailed explanation. 👍  with your solution and we'll have to revisit this later.